permissions/plugin-ws-continuous-testing-manager.yml

```diff
@@ -0,0 +1,9 @@
+---
+name: "ws-continuous-testing-manager"
+github: "jenkinsci/ws-ctm-plugin"
+paths:
+- "org/jenkins-ci/plugins/ws-ctm-plugin"
```

Comment:
not sure why you added this file, the other one covers what you need, and it has the correct path whereas this one has `-plugin` appended when it shouldn't

Reply:
@timja good catch.  I apologize.   I didn't notice the other commits.   I'll abort and clean this up and issue another pull request.

Question:  After this since this is a gradle plugin - I was a bit confused on the next steps.   If I want to manually do a release of our first plugin what is the last requirement after these permissions ?   The documentation is heavily focused on maven.

Thank you

Reply:
Not sure check the gradle plugin docs, you can only manually release it currently / setup your own custom CD for it.

very few plugins use gradle and no one has invested the time to get gradle working on this